README.md

```diff
@@ -1,14 +1,42 @@
-# Pollen
+# Pollen Operator
 
-More information: https://charmhub.io/pollen
+A juju charm deploying and managing Pollen on a machine. Pollen is a scalable, high performance, free software (AGPL) web server, that provides small strings of entropy, over TLS-encrypted HTTPS or clear text HTTP connections. You might think of this as ‘Entropy-as-a-Service’.
 
-Pollen is a scalable, high performance, free software (AGPL) web server, that
-provides small strings of entropy, over TLS-encrypted HTTPS or clear text HTTP
-connections. You might think of this as 'Entropy-as-a-Service'. This charm
-deploys a Pollen server.
+Pollinate is a free software (GPLv3) script that retrieves entropy from one or more Pollen servers and seeds the local Pseudo Random Number Generator (PRNG). You might think of this as PRNG-seeding via Entropy-as-a-Service.
 
-## Other resources
+This charm simplifies initial deployment and operations of Pollen on a machine, such as scaling the number of instances and clustering and more. As such, the charm makes it easy for those looking to take control of their own entropy generation whilst keeping operations simple.
 
-- [The Pollen source repository](https://github.com/canonical/pollen)
+For DevOps or SRE teams this charm will make operating Pollen simple and straightforward through Juju’s clean interface. It will allow easy deployment into multiple environments for testing of changes, and supports scaling out for enterprise deployments.
+Like any Juju charm, this charm supports one-line deployment, configuration, integration, scaling, and more.
+For Charmed Pollen, this includes:
+  - Scaling
 
-- See the [Juju SDK documentation](https://juju.is/docs/sdk) for more information about developing and improving charms.
+For information about how to deploy, integrate, and manage this charm,
+see the Official [Pollen Operator Documentation](https://charmhub.io/pollen/docs).
+
+## Get started
+
+You can follow the tutorial [here](https://charmhub.io/pollen/docs/how-to-relate-to-cos).
+
+You can check out the [full list of actions here](https://charmhub.io/pollen/actions).
+
+## Integrations
+
+This charm can be integrated with other Juju charms and services:
+
+  - [Grafana Agent](https://charmhub.io/grafana-agent-k8s): Grafana Agent is an OpenTelemetry Collector distribution with configuration inspired by Terraform.
+  - [Apachi2](https://charmhub.io/apache2): The Apache HTTP Server Project is an effort to develop and maintain an open-source HTTP server for modern operating systems.
+
+For a full list of integrations, see the [Charmhub documentation](https://charmhub.io/pollen/integrations).
+
+## Learn more
+* [Read more](https://charmhub.io/pollen) <!--Link to the charm's official documentation-->
+* [Developer documentation](https://github.com/canonical/pollen) <!--Link to any developer documentation-->
+* [Official webpage](https://github.com/canonical/pollen) <!--(Optional) Link to official webpage/blog/marketing content-->
```

Comment:
Is the original source here? https://github.com/dustinkirkland/pollen

Reply:
You have the original source but the Canonical source is a couple commits away and this can make differences.

Reply:
Ah ok, thanks for the context! No change needed for the time being